_bookdown.yml

```diff
@@ -31,7 +31,7 @@ rmd_files:
   - unit.Rmd
   - verify.Rmd
   - integrate.Rmd
-  - project.Rmd
+  - rse-project.Rmd
```

Comment:
Don't forget adding this to the Makefile too :)

Reply:
Shoot...is this new?  I haven't done it for any file...

Reply:
No worries, we'll fix any that are strays. I think I've fixed a few already, so might not be a problem! :)